elixir/apps/web/lib/web/components/navigation_components.ex

```diff
@@ -337,7 +337,7 @@ defmodule Web.NavigationComponents do
   def website_link(assigns) do
     ~H"""
     <.link
-      navigate={"https://www.firezone.dev#{@href}?utm_source=product"}
+      navigate={"https://www.firezone.dev?utm_source=product#{@href}"}
```

Comment:
I don't know if this is the right code to change the issue I saw, I took a guess.

I saw the Gateway deploy page links to e.g. `https://www.firezone.dev/kb/deploy/gateways#firewall-considerations?utm_source=product` but that means the `utm_source` tag is in the URL hash, which does not get sent to the server. I'm not sure if we pick those up in JS or server-side?